front/components/Interpellate/ButtonBackAndForth.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from 'next/link';
+
+import { buttonVariants } from '@/components/ui/button';
+import { ChevronLeft, ChevronRight } from 'lucide-react';
+
+type ButtonBackAndForth = {
+  linkto: string;
+  direction: 'back' | 'forth';
+  children: React.ReactNode;
+};
+export default function ButtonBackAndForth({ linkto, direction, children }: ButtonBackAndForth) {
+  return (
+    <Link
+      href={linkto}
+      className={buttonVariants({
+        variant: 'outline',
+        className: 'min-w-[200] bg-black text-white',
+      })}
+    >
+      {direction === 'back' ? (
+        <>
+          <ChevronLeft />
+          {children}
+        </>
+      ) : (
+        <>
+          {children}
+          <ChevronRight />
+        </>
+      )}
+    </Link>
+  );
+}
```

Comment:
Par consequent, ce serait plutot `ProgressButtons`

```suggestion
import Link from 'next/link';

import { buttonVariants } from '@/components/ui/button';
import { ChevronLeft, ChevronRight } from 'lucide-react';

type ButtonProps = {
  link: string;
  label: string;
};

type ProgressButtonsProps = {
  props1: ButtonProps;
  props2: ButtonProps;
};

export default function ProgressButtons({ props1, props2 }: ProgressButtonsProps) {
  const className = buttonVariants({
    variant: 'outline',
    className: 'min-w-[200] bg-black text-white',
  });

  return (
    <div className='my-12 flex justify-center gap-4'>
      <Link href={props1.link} className={className}>
        <ChevronLeft />
        {props1.label}
      </Link>
      <Link href={props2.link} className={className}>
        {props2.label} <ChevronRight />
      </Link>
    </div>
  );
}

```